src/ex7/client/public/index.html

```diff
@@ -9,7 +9,6 @@
       name="description"
```

Comment:
it didn't compiled for me out of the box, make sure you add every package to your package.json

![image](https://user-images.githubusercontent.com/65164924/178752311-27974bc4-0b22-4e75-815e-1b3946924de7.png)